CMP1903_A1_2324/Program.cs

```diff
@@ -10,10 +10,18 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            /*
-             * Create a Game object and call its methods.
-             * Create a Testing object to verify the output and operation of the other classes.
-             */
+            //Create a Game object and call its methods 
+            Game game = new Game();
+            int total = game.RollDice();
```

Comment:
'int total = game.RollDice();'
By calling the RollDice function in the constructor of the Game class, you could remove this step.